packages/opc-styles/test/playground.scss

```diff
@@ -0,0 +1,42 @@
+@import "../opc-styles.scss";
+
+
+// Expected input in scss
+
+// my-button {
+//   background-color: #{opc-local(BackgroundColor, fallback, region)}
+// }
+
+// Expected output in CSS
+
+// my-button {
+//   background-color: var(--opc-button--BackgroundColor, #E7F1FA); }
+
+
+// Temp variable to run opc-local function
+// see the documentation in function/_custom-properties.scss
+$LOCAL: button;
+
+
+op-button.soft {
+  background-color: #{opc-get-var(BackgroundColor, $opc-color--blue-50, soft)};
+  color: #{opc-get-var(TextColor, $opc-color--blue-400, soft)};
+  border-color: #{opc-get-var(BorderColor, $opc-color--blue-400, soft)};
+}
+
+op-button.flat {
+  background-color: #{opc-get-var(BackgroundColor, $opc-color--blue-50, flat)};
+  color: #{opc-get-var(TextColor, $opc-color--blue-400, flat)};
+  border-color: transparent;
+}
+
+// Output results
+// op-button.soft {
+//   background-color: var(--opc-button__soft--BackgroundColor, #E7F1FA);
+//   color: var(--opc-button__soft--TextColor, #0066CC);
+//   border-color: var(--opc-button__soft--BorderColor, #0066CC); }
+
+// op-button.flat {
+//   background-color: var(--opc-button__flat--BackgroundColor, #E7F1FA);
+//   color: var(--opc-button__flat--TextColor, #0066CC);
+//   border-color: transparent; }
```

Comment:
Is this required in the file? Should be in the docs rather than in the source IMO.

Reply:
@gisumit i agree with Mayur. All these are examples and should exist either in README.md usage/styles section

Reply:
adding an example to readme file.